go.mod

```diff
@@ -27,3 +27,5 @@ require (
 	golang.org/x/crypto v0.0.0-20190513172903-22d7a77e9e5f
 	google.golang.org/grpc v1.21.0 // indirect
 )
+
+go 1.13
```

Comment:
This keeps showing up in my commits to. Should we define go version explicitly in `go.mod`?

Reply:
Yes, good catch!

Sadly, it seems that Go will always bump it to the newer version. I am using 1.13 locally and therefore compiler will always upgrade it to 1.13. This is not great. I will check if there is a way to avoid this.